<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="AJAMPANA-IN" name="com.tc.groupcustomer.GC06_GroupCustomer3" tests="9" failures="1" timestamp="06 Jul 2020 12:25:40 GMT" time="2142.861" errors="0">
  <testcase name="TC27724" time="49.409" classname="com.tc.groupcustomer.GC06_GroupCustomer3"/>
  <testcase name="TC11" time="69.068" classname="com.tc.groupcustomer.GC06_GroupCustomer3"/>
  <testcase name="TC27723" time="684.425" classname="com.tc.groupcustomer.GC06_GroupCustomer3"/>
  <testcase name="TC18731" time="0.000" classname="com.tc.groupcustomer.GC06_GroupCustomer3">
    <skipped/>
  </testcase> <!-- TC18731 -->
  <testcase name="TC18730" time="574.355" classname="com.tc.groupcustomer.GC06_GroupCustomer3">
    <failure type="java.lang.AssertionError" message="Plans Name Mismatch expected [Auto_Plan5972] but found [Auto_Plan597]">
      <![CDATA[java.lang.AssertionError: Plans Name Mismatch expected [Auto_Plan5972] but found [Auto_Plan597]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at com.tc.groupcustomer.GC06_GroupCustomer3.TC18730(GC06_GroupCustomer3.java:308)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC18730 -->
  <testcase name="TC04" time="74.444" classname="com.tc.groupcustomer.GC06_GroupCustomer3"/>
  <testcase name="TC10" time="84.895" classname="com.tc.groupcustomer.GC06_GroupCustomer3"/>
  <testcase name="TC27725" time="54.644" classname="com.tc.groupcustomer.GC06_GroupCustomer3"/>
  <testcase name="TC07" time="551.621" classname="com.tc.groupcustomer.GC06_GroupCustomer3"/>
</testsuite> <!-- com.tc.groupcustomer.GC06_GroupCustomer3 -->
